reaper/reaper/src/main/kotlin/com/emergetools/reaper/CountingHashTracker.kt

```diff
@@ -8,7 +8,7 @@ class CountingHashTracker(private var onFlush: (Collection<Long>) -> Unit) : Has
     count++
   }
 
-  override fun flush() {
+  override fun flush(onFlush: (Collection<Long>) -> Unit) {
     onFlush(mutableSetOf(count))
```

Comment:
To avoid confusion here, should we remove the `onFlush` constructor param?

Reply:
oops, thanks!